liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/Dependency.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * (C) Copyright IBM Corporation 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.openliberty.tools.maven.applications;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.maven.plugins.annotations.Parameter;
+
+/**
+ * A Dependency specified with a filter and an optional location.
+ */
+public class Dependency {
+
+    /**
+     * Filter specifying the dependency to copy. The format is groupId:artifactId:version, with artifactId and 
+     * version optional. Maven dependency management will be used to resolve the dependency.
+     */
+    @Parameter(property = "filter")
```

Comment:
This and location `@Parameter` doesn't look right but I'm fuzzy on this. If you did -Dfilter=xyz from the CLI how would maven know which dependency to parameterize?  Would it do all of them?  Or are you thinking it's helpful if you happen to just have 1?

Reply:
I don't think supporting CLI properties for these makes sense. I'll change this to remove the property name.